File: src/main/java/dev/daae/time/controllers/StatusController.java
package dev.daae.time.controllers;

import dev.daae.time.services.StatusService;
import lombok.RequiredArgsConstructor;
import org.springframework.web.bind.annotation.*;

@RestController
@RequestMapping("/status")
@RequiredArgsConstructor
public class StatusController {

    private final StatusService statusService;

    @GetMapping("/current")
    public String currentStatus() {
        return statusService.currentStatus();
    }

    @GetMapping("/week")
    public String currentWeek() {
        return statusService.weekStatus();
    }

    @GetMapping("/today")
    public String today() {
        return statusService.todayStatus();
    }
}

File: src/main/kotlin/dev/daae/time/controllers/StatusController.kt
package dev.daae.time.controllers

import dev.daae.time.services.StatusService
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/status")
class StatusController(
    private val statusService: StatusService,
) {
    @GetMapping("/current")
    fun currentStatus() = statusService.currentStatus()

    @GetMapping("/week")
    fun currentWeek() = statusService.weekStatus()

    @GetMapping("/today")
    fun today(): String = statusService.todayStatus()
}
